### ISSUE TITLE
Creating Plans using Seeders

### ISSUE THREAD

**Issue by calle284** [rinvex/laravel-subscriptions#183]
Good day,

I am trying to create plans on migration but I keep getting slug issues (I think).

ERROR:
	The error that is happening here is that it keeps hitting the unique validation error whenever I reach the 'Free Yearly' as shown in the picture below.

![image](https://user-images.githubusercontent.com/6089768/151383517-7d935b0a-5276-421e-ad83-f4b1fccc7fd7.png)

No matter what I tried (listed below), when it reaches the 'Free Yearly', it still shows the error above.

![image (1)](https://user-images.githubusercontent.com/6089768/151383727-54fa5b67-b8dc-48cf-93c9-a754548d41f1.png)

WHAT I HAVE TRIED:

	- change all names to plans (from plan 1 to plan 6). Error still shows as in image above.
	
	- published migrations
		=> removed $table->unique('slug');
		
	- in all src/Models in the vendor
		=> removed all |unique:'.config('rinvex.subscriptions.tables.plans').',slug' from all models
	
	- Added 'allowDuplicateSlugs' to getSlugOptions()
		from return SlugOptions::create()
			  ->doNotGenerateSlugsOnUpdate()
			  ->generateSlugsFrom('name')
			  ->saveSlugsTo('slug');
		to  return SlugOptions::create()
			  ->doNotGenerateSlugsOnUpdate()
			  ->generateSlugsFrom('name')
			  ->saveSlugsTo('slug')
			  ->allowDuplicateSlugs();
	
	- Changed ->generateSlugsFrom('name') in getSlugOptions()
		from ->generateSlugsFrom('name')
		to  ->generateSlugsFrom(['name', 'invoice_interval'])

WHAT WORK BUT WITH ERRORS

	- the only thing that worked was to remove the validation check from trait and add slug in the create
		=> $freeMonthyPlan = app('rinvex.subscriptions.plan')->create ([
            'name'              => 'Free Monthly',
            'slug'       		=> 'MY-OWN-SLUG',
			...
but the error is that when adding features, there is slug issues as it is not auto added
	
Please help :/
Below is the planSeeder that I created

	<?php

	namespace Database\Seeders;
	use Illuminate\Database\Seeder;

	class PlanTableSeeder extends Seeder {
		public function run() {
			$freeMonthyPlan = app('rinvex.subscriptions.plan')->create ([
				'name'              => 'Free Monthly',
				'description'       => 'Free Plan',
				'price'             => 0.00,
				'signup_fee'        => 0.00,
				'invoice_period'    => 1,
				'invoice_interval'  => 'month',
				'trial_period'      => 0,
				'trial_interval'    => 'day',
				'sort_order'        => 1,
				'currency'          => 'BZE',
			]);
			
			// // Create multiple plan features at once
			// $freeMonthyPlan->features ()->saveMany ([
			//     new PlanFeature([
			//         'name'                  => 'listing_duration_days',
			//         'value'                 => 30,
			//         'sort_order'            => 10,
			//         // 'resettable_period'     => 1,
			//         // 'resettable_interval'   => 'month'
			//     ]),
			// ]);

			$standardMonthyPlan = app('rinvex.subscriptions.plan')->create ([
				'name'              => 'Standard Monthly',
				'description'       => 'Standard Plan',
				'price'             => 150.00,
				'signup_fee'        => 0.00,
				'invoice_period'    => 1,
				'invoice_interval'  => 'month',
				'trial_period'      => 0,
				'trial_interval'    => 'day',
				'sort_order'        => 2,
				'currency'          => 'BZE',
			]);
			
			$proMonthyPlan = app('rinvex.subscriptions.plan')->create ([
				'name'              => 'Pro Monthly',
				'description'       => 'Pro Plan',
				'price'             => 250.00,
				'signup_fee'        => 0.00,
				'invoice_period'    => 1,
				'invoice_interval'  => 'month',
				'trial_period'      => 0,
				'trial_interval'    => 'day',
				'sort_order'        => 3,
				'currency'          => 'BZE',
			]);

			$freeYearlyPlan = app('rinvex.subscriptions.plan')->create ([
				'name'              => 'Free Yealy',
				'description'       => 'Free Plan',
				'price'             => 0.00,
				'signup_fee'        => 0.00,
				'invoice_period'    => 1,
				'invoice_interval'  => 'year',
				'trial_period'      => 0,
				'trial_interval'    => 'day',
				'sort_order'        => 4,
				'currency'          => 'BZE',
			]);

			$standardYearlyPlan = app('rinvex.subscriptions.plan')->create ([
				'name'              => 'Standard Yealy',
				'description'       => 'Standard Plan',
				'price'             => 1650.00,
				'signup_fee'        => 0.00,
				'invoice_period'    => 1,
				'invoice_interval'  => 'year',
				'trial_period'      => 0,
				'trial_interval'    => 'day',
				'sort_order'        => 5,
				'currency'          => 'BZE',
			]);
			
			$proYearlyPlan = app('rinvex.subscriptions.plan')->create ([
				'name'              => 'Pro Yealy',
				'description'       => 'Pro Plan',
				'price'             => 2750.00,
				'signup_fee'        => 0.00,
				'invoice_period'    => 1,
				'invoice_interval'  => 'year',
				'trial_period'      => 0,
				'trial_interval'    => 'day',
				'sort_order'        => 6,
				'currency'          => 'BZE',
			]);
		}
	}

**Comment by Omranic** [comment]
Have you tried to add a `slug` to your plan?

**Comment by calle284** [comment]
yes I did. and it still gave the same error as if it was cached but I cleared cache too.

I gave all of them slugs
    free-montly
    standard-montly
    pro-montly
    free-yearly
    standard-yearly
    pro-yearly

**Comment by calle284** [comment]
> Have you tried to add a `slug` to your plan?


yes I did. and it still gave the same error as if it was cached but I cleared cache too.

see *WHAT WORK BUT WITH ERRORS* above

I gave all of them slugs
    free-montly
    standard-montly
    pro-montly
    free-yearly
    standard-yearly
    pro-yearly

**Comment by calle284** [comment]
@Omranic Can I please get an answer
